docs/source/models/causes/neonatal/index.rst

```diff
@@ -199,47 +199,61 @@ unit time.
 Modeling Strategy
 +++++++++++++++++
 
-The neonatal death model requires only the probability of death (aka "mortality risk") for the early and late neonatal time periods.  But computing this for an individual simulant is a bit complicated.  It will follow the pattern from the general mortality component in :code:`vivarium_public_health`, and work in rate space to make the math simpler.  The final step will be converting from rates to risks:
+The neonatal death model requires only the probability of death (aka "mortality risk") for the early and late neonatal time periods. Rather than using GBD mortality rates and converting them into probability of deaths, we will use mortality risk as direct input data into our model. We will calculate mortality risk input data as age-specific death counts divided by live birth counts from GBD.
+
+Note that this strategy does not require any conversion between rates to probabilities NOR does it require any scaling to the duration of the age group. The mortality risk calculated as described below already represents the probability of dying within a neonatal age group and can be used directly as such in the simulation.
+
+To avoid confusion with mortality *rates* (typically referred to as the all-cause mortality rate, ACMR, or cause-specific mortality rates, CSMRs), we will refer to mortality *risk* as AMR (all mortality risk) and CMR (cause mortality risk), where:
 
 .. math::
-    \begin{align*}
-    \text{mr}_i &= 1-\exp(-\text{ACMR}_i \cdot \Delta t),
-    \end{align*}
 
-where :math:`\text{mr}_i` is the probability of mortality for simulant :math:`i` during the early or late neonatal period, :math:`\text{ACMR}_i` is the all-cause mortality rate for the early or late neonatal period, and :math:`\Delta t` is length of that period in years.
+  AMR_{ENN} = \frac{\text{deaths due to all causes in the ENN age group}}{\text{live births}}
 
-The calculation of :math:`\text{ACMR}_i` is a bit complicated, however. We begin with a population ACMR and use the LBWSG PAF to derive a risk-deleted ACMR to which we can then apply the relative risk of LBWSG matching any risk exposure level.  Mathematically this is achieved by the following formula:
+  AMR_{LNN} = \frac{\text{deaths due to all causes in the LNN age group}}{\text{live births} - \text{deaths due to all causes in the ENN age group}}
```

Comment:
In the "Data values and sources" table below, you refer to these as `enn_amr` and `lnn_emr`. I think we should pick one of the two notations and stick with it. If you want to use the notation here, it might also be good to make the font consistent with the other uses of "AMR" and "CMR" by doing `text{AMR}_\text{ENN}` and `text{AMR}_\text{LNN}`.